src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -485,6 +485,14 @@ internal bool ReportDiagnostics(IEnumerable<Diagnostic> diagnostics, TextWriter
         {
             bool hasErrors = false;
             foreach (var diag in diagnostics)
+            {
+                ReportDiagnostic(diag);
+            }
+
+            return hasErrors;
+
+            // Local functions
+            void ReportDiagnostic(Diagnostic diag)
```

Comment:
>ReportDiagnostic [](start = 17, length = 16)

In Compilers code our convention is to start names of local functions with a small letter.

Reply:
Sure, will change.